modules/home-manager/gtk.nix

```
@@ -121,7 +122,7 @@ in
     (mkIf enable {
       gtk.theme =
         let
-          gtkTweaks = concatStringsSep "," cfg.tweaks;
+          gtkTweaks = if cfg.tweaks == [ ] then "default" else (concatStringsSep "," cfg.tweaks);
         in
         {
           name = "catppuccin-${cfg.flavor}-${cfg.accent}-${cfg.size}+${gtkTweaks}";
```

Following the changes from the nixpkgs PR, if tweaks is an empty list, the name will become:

```
"catppuccin-${cfg.flavor}-${cfg.accent}-${cfg.size}"
```

In which case we'd need to do something like this:

```suggestion
          gtkTweaks = if cfg.tweaks == [ ] then "" else (concatStrings [ "+" (concatStringsSep "," cfg.tweaks) ]);
        in
        {
          name = "catppuccin-${cfg.flavor}-${cfg.accent}-${cfg.size}${gtkTweaks}";
```